tasks/__init__.py

```diff
@@ -47,6 +47,7 @@
 ns.add_task(audit_tag_impact)
 ns.add_task(e2e_tests)
 ns.add_task(make_kitchen_gitlab_yml)
+ns.add_task(generate)
```

Comment:
I was wondering if `agent:generate` wasn't a better invoke target, but I realized, also from your changes, that generate may be required by other tasks in the main collection too, so I guess putting it here is the sensible thing to do.